Cargo.toml

```diff
@@ -34,9 +35,8 @@ html5ever = "0.26"
 kuchikiki = "0.8"
 lazy-regex = "3"
 mime2ext = "0.1"
-mustache = "0.9"
 once_cell = "1"
-pling = "0.3"
+pling = { git = "https://github.com/EdJoPaTo/pling", branch = "main", default-features = false, features = ["clap", "reqwest"] }
```

Comment:
Why was this change made?

Reply:
Because pling 0.4 is not released yet and the current main branch is used. Pling 0.4 needs to be released and used here before merging.